cloud/src/common/config.h

```diff
@@ -264,6 +264,7 @@ CONF_Bool(enable_loopback_address_for_ms, "false");
 
 // delete_bitmap_lock version config
 CONF_mString(use_delete_bitmap_lock_version, "v1");
+CONF_mInt64(update_delete_bitmap_lock_white_list_interval_seconds, "10");
```

Comment:
this is redundant interval, we should detect whitelist change when we try to get version,
once it is changed (we can store an original sting in DeleteBitmapWhitelist for checking) update the whitelist map